build-system/server/app-video-testbench.js

```diff
@@ -367,7 +367,7 @@ function isValidExtension(extension) {
 
 
 function runVideoTestBench(req, res, next) {
-  fs.readFileAsync(sourceFile).then(contents => {
+  fs.promises.readFile(sourceFile).then(contents => {
```

Comment:
Would it make sense to `const fs = require('fs').promises` and use `fs` normally?

Reply:
No, because I want it to be explicit that `fs` is the result of `require('fs')`. This also allows for the use of non-promise methods via `fs`.

Reply:
:+1: